### DISCUSSION TITLE
Can you explain the effect of the data domain ID?

### DISCUSSION THREAD

**Discussion by lars76** [sarulab-speech/UTMOSv2#3] [General]
In https://github.com/sarulab-speech/UTMOSv2/blob/main/docs/inference.md you offer the option of performing a prediction with different dataset IDs. Do the different dataset IDs refer to differently trained models or do they differ by test-time augmentation? Which dataset ID should I use in which case? Suppose I have a non-English audio dataset, which dataset ID would be the best?

**Comment by kAIto47802** [comment]
@lars76 
Thank you for your question. Sorry for the unclear explanation, as this is a GitHub release ahead of our paper publication.
We have incorporated an encoding to represent data domain IDs in order to handle datasets consisting of multiple MOS tests that may contain range-equalizing bias (Cooper+23). The `--predict_dataset` option indicates which data domain ID to use during inference, i.e., it specifies within which dataset's MOS test values the predictions should be assumed.
Also, since UTMOSv2 was developed for the VoiceMOS Challenge 2024 Track 1, only English audio datasets were used for training, and performance on other languages has not been validated.

**Comment by maevdokimov** [comment]
Hi
I assume the model is biased towards the source dataset distribution. So if I take some arbitrary data in English, will Sarulab Data be the best dataset id out of the box?

**Comment by kAIto47802** [comment]
@maevdokimov 
Thank you for your question. Due to the range-equalizing bias (Cooper+23), what we define as "best" depends on the intended target. However, if the goal is the evaluation of high-quality synthetic speech, the sarulab-data is the most appropriate dataset ID. I apologize for the current confusion as the paper has not yet been published, but within the paper, we have thoroughly investigated the impact of differences between each dataset ID, and the detailed results are provided there. The paper is expected to be published in several days.
